src/java.base/share/classes/java/util/concurrent/CopyOnWriteArrayList.java

```diff
@@ -262,7 +264,7 @@ public int indexOf(@Nullable Object o) {
      *         {@code -1} if the element is not found.
      * @throws IndexOutOfBoundsException if the specified index is negative
      */
-    public int indexOf(E e, int index) {
+    public int indexOf(@Nullable E e, int index) {
```

Comment:
`@Nullable` here is pretty weird. It looks well justified to me by the contract (and implementation). However, I'm not sure how often callers would _want_ to pass `null` here. I'm not sure if you'd want to leave `@Nullable` off for that reason.